main/nav_create_simulated_model.py

```
@@ -0,0 +1,1739 @@
+#!/usr/bin/env python3
+import json
+import re
+import sys
+from typing import Any, Callable, Optional, cast
```

Ruff (0.14.6)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)